services/command-router-base/src/test/java/org/eclipse/hono/commandrouter/impl/kafka/KafkaBasedMappingAndDelegatingCommandHandlerTest.java

```diff
@@ -98,21 +100,25 @@ public void setUp() {
                 .thenReturn(Future.succeededFuture(createTargetAdapterInstanceJson(deviceId, adapterInstanceId)));
 
         internalCommandSender = mock(KafkaBasedInternalCommandSender.class);
-        when(internalCommandSender.sendCommand(any(), any())).thenReturn(Future.succeededFuture());
+        when(internalCommandSender.sendCommand(
+                any(CommandContext.class),
+                anyString()))
+            .thenReturn(Future.succeededFuture());
 
         final KafkaBasedCommandResponseSender kafkaBasedCommandResponseSender = mock(KafkaBasedCommandResponseSender.class);
 
         vertx = mock(Vertx.class);
         final Context context = VertxMockSupport.mockContext(vertx);
         final KafkaCommandProcessingQueue commandQueue = new KafkaCommandProcessingQueue(context);
         final CommandRouterMetrics metrics = mock(CommandRouterMetrics.class);
+        when(metrics.startTimer()).thenReturn(Timer.start());
         final Tracer tracer = TracingMockSupport.mockTracer(TracingMockSupport.mockSpan());
         cmdHandler = new KafkaBasedMappingAndDelegatingCommandHandler(vertx, tenantClient, commandQueue, commandTargetMapper,
                 internalCommandSender, kafkaBasedCommandResponseSender, metrics, tracer);
     }
 
     /**
-     * Verifies the behaviour of the 
+     * Verifies the behavior of the 
```

Comment:
_behaviour_  is also correct. Behaviour (UK) and behavior (US).

Reply:
I just changed it because my Eclipse IDE uses US English spell checking and always marked it as wrong ;-)